test/swfs/as3-loader/events/loader-events.swf.trace

```diff
@@ -0,0 +1,36 @@
+constructed root1
+added root1
+added instance1
+added instance3
+exitFrame 1
+root loader: init 1293
+root loader: progress 1293
+root loader: complete 1293
+enterFrame 2
+run frame 2 in root1
+exitFrame 2
+child 1 loader: open 0
+child 1 loader: progress 0
+child 1 loader: progress 8289
+child 2 loader: open 0
+child 2 loader: progress 0
+child 2 loader: progress 8289
+enterFrame 3
+added instance5
+construct instance2, parent: [object Loader] (TODO: parent should be null at this point, and the previous ADDED not dispatched)
+added instance2
+added instance6
+construct instance4, parent: [object Loader] (TODO: parent should be null at this point, and the previous ADDED not dispatched)
```

Comment:
Please provide test cases with comments like this. You clearly created a test case that shows this, so it doesn't make sense for me to try to figure out the details of it all over again.